utilities/dockerfiles/wildfly/Dockerfile

```
@@ -0,0 +1,46 @@
+RUN cd $HOME
+RUN yum install -y epel-release
+RUN yum install -y net-tools
+RUN yum install -y curl cabextract xorg-x11-font-utils fontconfig
+RUN curl -O http://li.nux.ro/download/nux/dextop/el7/x86_64/nux-dextop-release-0-5.el7.nux.noarch.rpm
+RUN rpm -Uvh nux-dextop-release-0-5.el7.nux.noarch.rpm
+RUN rpm -Uvh  https://downloads.sourceforge.net/project/mscorefonts2/rpms/msttcore-fonts-installer-2.6-1.noarch.rpm
+RUN rm nux-dextop-release-0-5.el7.nux.noarch.rpm
+
+# Add the WildFly distribution to /opt, and make wildfly the owner of the extracted tar content
+# Make sure the distribution is available from a well-known place
+RUN cd /tmp \
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)